pytket/extensions/qiskit/backends/ibm.py

```diff
@@ -96,7 +94,7 @@
 from .config import QiskitConfig
 
 if TYPE_CHECKING:
-    from qiskit_ibm_provider.ibm_backend import IBMBackend as _QiskIBMBackend  # type: ignore
+    from qiskit.providers.backend import BackendV1  # type: ignore
```

Comment:
Have you taken a look on the difference of V1 and V2? Do we want to upgrade to V2?

Reply:
Yes, I will do this as part of #19 .